### ISSUE TITLE
Cannot navigate with VoiceOver

### ISSUE THREAD

**Issue by egdegauperaa** [janosh/svelte-multiselect#118]
I noticed during testing that VoiceOver:

- Doesn't recognize that a list has been opened, only the input field.
- Doesn't recognize that when entering text the list is updated
- Doesn't recognizeNavigating the dropdown list with the arrows is not reported either

I might be able to contibute code to fix this.

**Comment by janosh** [comment]
Thanks for pointing this out. Any improvements in this regard are very welcome.
